cmd/check.go

```diff
@@ -243,14 +244,43 @@ func check() {
 			}
 			op, err = output.NewCLIOutput(outputFormat)
 		} else if outputType == "local" {
-			outputFormat := outputDefinition["format"].(string)
-			outputPath := expandHome(outputDefinition["path"].(string))
-			op, err = output.NewLocalOutput(outputFormat, outputPath)
+			outputFormat, ok := outputDefinition["format"].(string)
+			if !ok {
+				log.Fatal("Missing 'format' property in local output configuration.")
+			}
+			outputPath, ok := outputDefinition["path"].(string)
+			if !ok {
+				log.Fatal("Missing 'path' property in local output configuration.")
+			}
+			op, err = output.NewLocalOutput(outputFormat, expandHome(outputPath))
 		} else if outputType == "gcs" {
-			outputFormat := outputDefinition["format"].(string)
-			outputBucketName := outputDefinition["bucket-name"].(string)
-			outputCredentialsPath := outputDefinition["credentials-path"].(string)
+			outputFormat, ok := outputDefinition["format"].(string)
+			if !ok {
+				log.Fatal("Missing 'format' property in gcs output configuration.")
+			}
+			outputBucketName, ok := outputDefinition["bucket-name"].(string)
+			if !ok {
+				log.Fatal("Missing 'bucket-name' property in gcs output configuration.")
+			}
+			outputCredentialsPath, ok := outputDefinition["credentials-path"].(string)
+			if !ok {
+				log.Fatal("Missing 'credentials-path' property in gcs output configuration.")
+			}
 			op, err = gcs.NewOutput(ctx, outputFormat, outputBucketName, outputCredentialsPath)
+		} else if outputType == "azblob" {
+			outputFormat, ok := outputDefinition["format"].(string)
+			if !ok {
+				log.Fatal("Missing 'format' property in azblob output configuration.")
+			}
+			outputContainer, ok := outputDefinition["container"].(string)
+			if !ok {
+				log.Fatal("Missing 'container' property in azblob output configuration.")
+			}
+			accountName, accountKey := os.Getenv("AZURE_STORAGE_ACCOUNT"), os.Getenv("AZURE_STORAGE_ACCESS_KEY")
```

Comment:
Can we use a key file here like we do for GKE/AKS access?

Reply:
I looked into that briefly and didn't found an immediate well-known way of doing that, so I decided to use the env vars.

I can investigate that later.

PD: the token we use for AKS is not ideal. It is very short-lived and I haven't found a good way to extend its life. That is another thing to look into.

I have opened another issue to improve how we authenticate against Azure in general: #56